pkg/model/misc.go

```diff
@@ -0,0 +1,25 @@
+/*
```

Comment:
Just in general whenever a shortcut name like `s()` is used, I have found it useful to have it wrap a more verbose function - that's all I am getting at - simple code doc would work too